src/scripts/migrate.py

```diff
@@ -120,13 +123,15 @@ def slurp(
         else:
             next_mondo_id, mondo_term_ids = _get_next_available_mondo_id(next_mondo_id, max_id, mondo_term_ids)
             mondo_id = 'MONDO:' + str(next_mondo_id).zfill(7)  # leading 0-padding
+        qualified_parents = [p for p in t.direct_owned_parent_curies
+                             if p in match_types and match_types[p] in ['skos:exactMatch', 'skos:narrowMatch']]
+        qualified_mondo_paents = [mondo_id_map[p] for p in qualified_parents if p in mondo_id_map]
```

Comment:
Fixed [new bug](https://github.com/monarch-initiative/mondo-ingest/pull/316#discussion_r1200443120) where parents in robot template were not Mondo IDs.

Reply:
can we seperate this bugfix into new PR and merge asap? The testing thing will take a bit longer..

Reply:
Oh yeah of course. Good call.